### PR TITLE
{tools}[dummy] time 1.7

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/t/time/time-1.7.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+easyblock = 'ConfigureMake'
+
+name = 'time'
+version = '1.7'
+
+homepage = 'http://www.gnu.org/software/time/'
+description = """iThe `time' command runs another program, then displays information about the resources used by that
```
Comment (pforai) [inline easybuild/easyconfigs/t/time/time-1.7.eb]: Drop the i before "The".

Comment (boegel) [inline easybuild/easyconfigs/t/time/time-1.7.eb]: fixed

**Comment by pforai** [comment]
Test report by @pforai
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SLES 11_SP2, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/2d3b16234e051127dc65 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 0 out of 0 (1 easyconfigs in this PR)
Darwin UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz, Python 2.7.10
See https://gist.github.com/e265e8e4471f88d33dd6 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Darwin UNKNOWN UNKNOWN, Intel(R) Core(TM) i5-4258U CPU @ 2.40GHz, Python 2.7.10
See https://gist.github.com/c702df345fd599a2a41d for a full test report.

**Pull request merged**